_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title-short: "Snowplow R117 release"
+title: "Snowplow R117 release"
+tags: [snowplow, release]
+author: Łukasz Indykiewicz
+category: Releases
+permalink: /blog/2019/11/01/snowplow-r117-biskupin/
+---
+
+Snowplow release R117 is named after [Biskupin](https://en.wikipedia.org/wiki/Biskupin), which is "an archaeological site and a life-size model of a late Bronze Age fortified settlement in north-central Poland that also serves as a archaeological open-air museum".
+
+This release focuses on delivering small, but important improvements to Snowplow ecosystem:
+
+1. [TLS port binding and certificate](#tls-port-binding-and-certificate)
+2. [Referer parser refreshment](#referer-parser-refreshment)
+3. [IPv6 anonymization](#ipv6-anonymization)
+4. [Additional event fingerprint hashing methods (SHA1, SHA256, SHA384, SHA512)](#additional-event-fingerprint-hashing-methods)
+5. [Support to the spot market for core instances](#support-to-the-spot-market-for-core-instances)
+
```

Comment:
Roadmap and getting help should be in contents.

Reply:
I also added `Upgrading`